### PR TITLE
Created data layer intefaces

### PR DESCRIPTION
**Pull request by FilipZecevic**

### CODE REVIEW

File: src/Data/Intefaces/IFuelLog.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿namespace Data.Intefaces
+{
+    internal interface IFuelLog
```
Comment (injacmarko) [inline src/Data/Intefaces/IFuelLog.cs]: Internal?

File: src/Data/Intefaces/IVehicle.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿namespace Data.Intefaces
+{
+    internal interface IVehicle
```
Comment (injacmarko) [inline src/Data/Intefaces/IVehicle.cs]: Same here

File: src/BuisinessLogic/Intefaces/IFuelLog.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
+﻿namespace BuisinessLogic.Intefaces
+{
+    public interface IFuelLog
+    {
+        public interface IFuelLog
```
Comment (injacmarko) [inline src/BuisinessLogic/Intefaces/IFuelLog.cs]: Interface in an interface?

File: src/BuisinessLogic/Intefaces/IVehicle.cs [hunk 4]
```diff
@@ -0,0 +1,14 @@
+﻿namespace BuisinessLogic.Intefaces
+{
+    public interface IVehicle
```
Comment (injacmarko) [inline src/BuisinessLogic/Intefaces/IVehicle.cs:5]: here also

File: src/BuisinessLogic/Intefaces/IVehicle.cs [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    {
+        public interface IVehicle
+        {
+            IEnumerable<VehicleDTO> GetAllVehicles();
+            VehicleDTO GetVehicleById(int id);
+            void AddVehicle(Vehicle vehicle);
```
Comment (injacmarko) [inline src/BuisinessLogic/Intefaces/IVehicle.cs]: VehicleDTO

File: src/BuisinessLogic/Intefaces/IFuelLog.cs [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+        {
+            IEnumerable<FuelLogDTO> GetAllFuelLogs();
+            IEnumerable<FuelLogDTO> GetAllFuelLogsByVehicleId(int id);
+            FuelLogDTO GetFuelLogById(int id);
+            void AddFuelLogToVehicle(int id, FuelLog fuel_log);
```
Comment (injacmarko) [inline src/BuisinessLogic/Intefaces/IFuelLog.cs]: FuelLogDTO

**Review by injacmarko — CHANGES_REQUESTED** [review]

**Review by injacmarko — APPROVED** [review]
Looks good now

**Review by KoAleksandar — APPROVED** [review]

**Pull request merged**